pkg/sql/catalog/accessor.go

```diff
@@ -13,14 +13,13 @@ package catalog
 import (
 	"context"
 
-	"github.com/cockroachdb/cockroach/pkg/keys"
 	"github.com/cockroachdb/cockroach/pkg/kv"
-	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 )
 
-// Accessor provides access to sql object descriptors.
+// Accessor provides access to descriptors.
+// It is implemented by the collection.
```

Comment:
This documentation might be worth improving a bit.

Reply:
Done